4-moodle-image-builder/config.php.template

```diff
@@ -0,0 +1,35 @@
+
+$CFG->enable_read_only_sessions = true;
+
+$CFG->localcachedir = '/tmp/moodle/localcache';
+
+// $CFG->lock_factory = '\\local_redislock\\lock\\redis_lock_factory';
```

Comment:
Once the plugin about file lock using redis is installed, it is advisable to enable these settings by uncommenting it.

Reply:
Since this plugin is a local plugin, so far I could not find a way to install it automatically, link here:
https://github.com/open-lms-open-source/moodle-local_redislock